src/tz_aix.rs

```diff
@@ -0,0 +1,26 @@
+use std::fs::OpenOptions;
+use std::io::{BufRead, BufReader};
+use std::env;
+
+pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
+    env::var("TZ").map_err(|_| crate::GetTimezoneError::OsError)
```

Comment:
this will panic if `TZ` is not valid UTF-8.

this instead should do something like:

```rust
let timezone = env::var_os("TZ").ok_or(crate::GetTimezoneError::OsError)?;
timezone.into_string().map_err(|_| crate::GetTimezoneError::OsError)
```

Reply:
According to the docs, [`std::env::var`](https://doc.rust-lang.org/stable/std/env/fn.var.html) does not panic on a Unicode error. (And it looks like this has been the case since rust 1.0.0.)

Reply:
oh wow, I've had this wrong in my head for quite some time then. Sorry!